kitsune/sumo/static/sumo/js/tests/instant_search_tests.js

```diff
@@ -0,0 +1,140 @@
+import {default as mochaJsdom, rerequire} from 'mocha-jsdom';
+import {default as chai, expect} from 'chai';
+import React from 'react';
+import chaiLint from 'chai-lint';
+import sinon from 'sinon';
+
+import mochaK from './fixtures/mochaK.js';
+import mochaJquery from './fixtures/mochaJquery.js';
+import mochaGoogleAnalytics from './fixtures/mochaGoogleAnalytics.js';
+import mochaNunjucks from './fixtures/mochaNunjucks.js';
+import mochaGettext from './fixtures/mochaGettext.js';
+
+chai.use(chaiLint);
+
+describe('instant search', () => {
+  mochaJsdom({useEach: true});
+  mochaJquery();
+  mochaK();
+  mochaGoogleAnalytics();
+  mochaGettext();
+  mochaNunjucks();
+  /* globals window, document, $ */
+
+  describe('', () => {
```

Comment:
Um, scratch all that. I had a lot of trouble getting Nunjucks working, so I did a lot of stuff that didn't always end up helping. I guess this was one of them. I can rearrange the code a bit to remove the describe block, since it is confusing.